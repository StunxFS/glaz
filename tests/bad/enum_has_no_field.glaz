enum A {
  B
}

fn main() {
  let mut x = A::C;
}
