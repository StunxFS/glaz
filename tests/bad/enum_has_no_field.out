tests/bad/enum_has_no_field.glaz:6:19: error: enum `A` has no field `C`
     | 
   6 |   let mut x = A::C;
     |                  ^
glaz error: could not compile `enum_has_no_field`, aborting due to previous error.
